DatadogWebViewTracking/Sources/WebViewTracking.swift

```diff
@@ -4,55 +4,126 @@
  * Copyright 2019-Present Datadog, Inc.
  */
 
+import Foundation
 import DatadogInternal
 
-/// Defines methods to send WebView related information
-internal protocol WebViewTracking {
-    /// Sends a message
-    /// - Parameter message: The message to send
-    func send(message: WebViewTrackingMessage) throws
-}
+#if !os(tvOS)
+import WebKit
+#endif
 
-/// Datadog implementation of `WebViewTracking`
-/// - Note:
-/// Cross platform SDKs should instantiate this type with a `DatadogCoreProtocol` implementation
-/// and pass WebView related messages using the message bus of the core.
-internal struct WebViewTrackingCore: WebViewTracking {
-    enum MessageKeys {
-        static let browserLog = "browser-log"
-        static let browserRUMEvent = "browser-rum-event"
+/// Real User Monitoring allows you to monitor web views and eliminate blind spots in your hybrid iOS applications.
+///
+/// # Prerequisites:
+/// Set up the web page you want rendered on your mobile iOS and tvOS application with the RUM Browser SDK
+/// first. For more information, see [RUM Browser Monitoring](https://docs.datadoghq.com/real_user_monitoring/browser/#npm).
+///
+/// You can perform the following:
+/// - Track user journeys across web and native components in mobile applications
+/// - Scope the root cause of latency to web pages or native components in mobile applications
+/// - Support users that have difficulty loading web pages on mobile devices
+public final class WebViewTracking {
+#if !os(tvOS)
+    /// Enables SDK to correlate Datadog RUM events and Logs from the WebView with native RUM session.
+    ///
+    /// If the content loaded in WebView uses Datadog Browser SDK (`v4.2.0+`) and matches specified
+    /// `hosts`, web events will be correlated with the RUM session from native SDK.
+    ///
+    /// - Parameters:
+    ///   - webview: The web-view to track.
+    ///   - hosts: A set of hosts instrumented with Browser SDK to capture Datadog events from.
+    ///   - core: Datadog SDK core to use for tracking.
+    public static func enable(
+        webview: WKWebView,
```

Comment:
I think `webview` is not the right form, should be `webView` (it is `WKWebView` not `WKWebview`).